src/solidity-frontend/solidity_template.h

```diff
@@ -146,20 +147,21 @@ __ESBMC_HIDE:;
   return address_t(hash);
 }
 
-uint256_t _pow(unsigned int base, unsigned int exp) {
```

Comment:
Why don't you use our C math functions? https://github.com/esbmc/esbmc/tree/master/src/c2goto/library

Reply:
![image](https://github.com/user-attachments/assets/09790014-e5ba-47f7-a97c-9fbe5e71a1e5)

Yeah I believe we are using the c math now.